kythe/go/extractors/config/wrapper/gradle_build_modifier.go

```diff
@@ -0,0 +1,79 @@
+package wrapper
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"regexp"
+)
+
+// These are the lines necessary for gradle build to use a different javac.
+var kytheJavacWrapper = `
```

Comment:
I was ambivalent about whether to plop this into a datafile or just inline it here.